docs/GDPs/gdp-0001-standalone-cli.rst

```diff
@@ -0,0 +1,434 @@
+=================================
```

Comment:
General formatting comments:

- I recommend to use GTScript and Python capitalized spellings
- For coherency with code files, I would limit line length to 99 or 100 chars
- Bullet points should not be indented in restructured text
- For code references in the text, it's better to use either a doble backtick syntax (\`\`some code identifier\`\`) or the \`code\` role (:code:\`some code identifier\`). Just one backtick is not properly rendered on Github and I think it is not really meant to be used like that.